element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/HttpServerData.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.connector.generator.http;
+package io.camunda.connector.generator.dsl.http;
 
 /** Data for the HTTP base URL selector property of the generated element template. */
 public record HttpServerData(String baseUrl, String label) {}
```

Comment:
Nice idea to have make this available in the DSL. Lets see whether it can be used for other generators as well.